engine/table/src/test/java/io/deephaven/engine/table/impl/FuzzerTest.java

```diff
@@ -132,7 +132,9 @@ private void testFuzzerScriptFile(final long timeSeed, String s, boolean realtim
 
         final TimeTable timeTable = (TimeTable) session.getVariable("tt");
 
-        for (int step = 0; step < 100; ++step) {
+        final int steps = TstUtils.SHORT_TESTS ? 20 : 100;
+
+        for (int step = 0; step < steps; ++step) {
```

Comment:
From an architectural standpoint, I think everything that relies on `TstUtils.SHORT_TESTS` (or more generally, some arbitrary number of iterations/seeds for testing) is a "code smell", at least from the perspective of "unit tests" that guard PRs from merging. It may just be around the nomenclature we use, but my future wishlist would include rebrading and reconfiguring how we think about these types of "tests".

That said, I understand the practical nature of why it is the way it is, and that we want it to be faster - and I'm not proposing you fix our architectural shortcomings. At the very least, I'm happy that we'll have hard references to a lot of these places now that SHORT_TESTS is being introduced.